client/app/dbaas/logs/detail/streams/alerts/add/alerts-add.html

```diff
@@ -121,6 +121,15 @@
             </oui-numeric>
         </oui-field>
 
+        <oui-field data-label="{{ ::'add_alert_value_query_filter' | translate }}" data-size="xl"
+            data-help-text="{{ ::'add_alert_value_query_filter_description' | translate }}">
+            <input type="text" id="query_filter" name="query_filter"
```

Comment:
Prefer camel case please :)

Reply:
No! Prefer `query-filter` for instance.

Since we use the BEM naming convention on the OVH UI Kit, it make more sense to stay with lowercase and hyphens.